zio-http/src/main/scala/zio/http/service/ServerInboundHandler.scala

```diff
@@ -29,22 +29,28 @@ private[zio] final case class ServerInboundHandler[R](
         val req  = Request.fromFullHttpRequest(jReq)
         val exit = http.execute(req)
 
-        if (self.attemptFastWrite(exit)) {
-          Unsafe.releaseRequest(jReq)
-        } else
-          runtime.unsafeRun {
-            self.attemptFullWrite(exit, jReq) ensuring ZIO.succeed { Unsafe.releaseRequest(jReq) }
-          }
+        Unsafe.unsafe { implicit u =>
```

Comment:
Maybe it would be better to have an implicit unsafe for the whole class as it's implementing a Netty interface

Reply:
Agreed. You can also cheat and `import zio.Unsafe.unsafe` since it's in the `zio` package. 🤫